packages/strapi-design-system/src/Layout/HeaderLayout.js

```diff
@@ -81,9 +83,13 @@ export const BaseHeaderLayout = React.forwardRef(
                 <Typography variant="beta" as="h1" {...props}>
                   {title}
```

Comment:
I was wondering about allowing the title to be a node too. My only worry is that so far we make sure we have an `h1` on the page, which wouldn't be the case allowing a node, wdyt @gu-stav?

Reply:
I'd say the design-system should be as strict as possible. If we don't need to open up that slot for nodes, I'd keep it as is.